sdk/iOS/test/MSCoreDataStoreTests.m

```diff
@@ -47,6 +50,54 @@ -(void)testInit
     XCTAssertNotNil(self.store, @"store creation failed");
 }
 
+// Tests MSCoreDataStore Read, Update and Delete operations are case insensitive for the Id column.
+-(void)testReadUpdateDelete_CaseInsensitiveIdOperations
+{
+    NSError *error;
+
+    // Seed data to be inserted in the table
+    NSString * const originalItemId = @"itemid";
```

Comment:
Updated formatting to *const now.
We should try to be const correct for variables that have scope wider than a function. And looks like we have been doing it for global string literals in test as well as product code, which is good (try searching for static nsstring in test code).
Although using const for syncTable and query would be extremely ideal, it is ok to be lenient as it has a restricted scope (function) and less chances of being modified unintentionally. 
Also, IMHO using const more liberally can only be good.